.gitignore

```diff
@@ -28,4 +28,4 @@ data
 /*.db
 
 /nuts-node
-nuts.yaml
+./nuts.yaml
```

Comment:
if the goal is to include nuts.yaml in the e2e directory is not better to add `!nuts.yaml` in the e2e gitignore? (not sure if that actually works)

Reply:
I'll test it